source/features/forked-to.tsx

```diff
@@ -2,11 +2,10 @@ import './forked-to.css';
 import React from 'dom-chef';
 import cache from 'webext-storage-cache';
 import select from 'select-dom';
-import elementReady from 'element-ready';
-import * as pageDetect from 'github-url-detection';
-
 import ForkIcon from 'octicon/repo-forked.svg';
 import CheckIcon from 'octicon/check.svg';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
```

Comment:
These were specifically split in 2 groups because this feature uses 3 icons. Reverting